File: matlab/test/Ice/objects/CI.m
%{
**********************************************************************

Copyright (c) 2003-present ZeroC, Inc. All rights reserved.

**********************************************************************
%}

classdef CI < Test.C
    methods
        function ice_preMarshal(obj)
            obj.preMarshalInvoked = true;
        end

        function ice_postUnmarshal(obj)
            obj.postUnmarshalInvoked = true;
        end
    end
end

File: swift/test/Ice/objects/CI.swift
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

public class CI : C {

    override public func ice_preMarshal() {
        preMarshalInvoked = true;
    }

    override public func ice_postUnmarshal() {
        postUnmarshalInvoked = true;
    }
}
